resources/views/users/index.blade.php

```diff
@@ -0,0 +1,47 @@
+<x-app-layout>
+    <x-slot name="header">
+        <h2 class="font-semibold text-xl text-gray-800 dark:text-gray-200 leading-tight">
+            {{ __('User List') }}
+        </h2>
+    </x-slot>
+
+    <div class="py-12">
+        <div class="max-w-7xl mx-auto sm:px-6 lg:px-8">
+            <div class="bg-white dark:bg-gray-800 overflow-hidden shadow-sm sm:rounded-lg">
+                <div class="p-6 text-gray-900 dark:text-gray-100">
+                    {{ __("User List") }}
+                </div>
+            </div>
+            <x-primary-button class="mt-4 mb-4">
+                {{__('Create')}}
+            </x-primary-button>
+            <Table class="table">
+                <thead>
+                    <tr>
+                        <th class="text-gray-900 dark:text-gray-100 text-center" scope="col">#</th>
+                        <th class="text-gray-900 dark:text-gray-100 text-center" scope="col">Name</th>
+                        <th class="text-gray-900 dark:text-gray-100 text-center" scope="col">Username</th>
+                        <th class="text-gray-900 dark:text-gray-100 text-center" scope="col">Action</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    @foreach ($users as $index => $user)
+                    <tr>
+                        <th class="text-gray-900 dark:text-gray-100 text-center" scope="col">{{$index}}</th>
```

Comment:
```suggestion
                        <th class="text-gray-900 dark:text-gray-100 text-center" scope="col">{{ ++$index }}</th>
```
M suggest dùng ++$index để nó đếm từ 1 thay vì 0 nha.